ralph-wiggum-toolkit/recipes/greenfield/templates/PROMPT_build.md

```
@@ -0,0 +1,20 @@
+0a. Study `specs/*` with up to 500 parallel Sonnet subagents to learn the application specifications.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)